src/Illuminate/Pipeline/Pipeline.php

```diff
@@ -113,9 +120,30 @@ public function then(Closure $destination)
             array_reverse($this->pipes()), $this->carry(), $this->prepareDestination($destination)
         );
 
+        if ($this->catchCallback) {
+            try {
+                return $pipeline($this->passable);
+            } catch (\Throwable $th) {
+                return call_user_func($this->catchCallback, $th, $this->passable);
```

Comment:
What happens when a user did not provide a catch callback?

Reply:
If no callback is being provided, and the user is chaining the `catch` method, it would throw an `ArgumentCountError`. If the user does not chain the `catch` method, it would run as normal without the try catch block.

Reply:
you are right, I missed the if clause. sorry =)